tests/scripts/seviri.Natural_Color.imagery_clean.sh

```diff
@@ -11,8 +11,8 @@ geoips run single_source $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20250624/120
              --reader_name seviri_hrit \
              --product_name Natural_Color \
              --output_formatter imagery_clean \
-             --compare_path "$GEOIPS_PACKAGES_DIR/geoips/tests/outputs/seviri.Natural_Color.imagery_clean" \
-             --sector_list global_cylindrical \
+             --compare_path "$GEOIPS_PACKAGES_DIR/geoips/tests/outputs/seviri.Natural-Color.imagery_clean" \
```

Comment:
Oh shoot.  I think we *want* these to be Natural-Color, but they are and should be Natural_Color right now. I don't know how this got switched back again